core/src/main/scala/org/apache/spark/metrics/sink/ConsoleSink.scala

```diff
@@ -25,27 +25,29 @@ import com.codahale.metrics.{ConsoleReporter, MetricRegistry}
 import org.apache.spark.SecurityManager
 import org.apache.spark.metrics.MetricsSystem
 
-private[spark] class ConsoleSink(val property: Properties, val registry: MetricRegistry,
-    securityMgr: SecurityManager) extends Sink {
+private[spark] class ConsoleSink(
+    property: Properties,
+    registry: MetricRegistry,
+    securityMgr: SecurityManager) extends Sink(property, registry) {
   val CONSOLE_DEFAULT_PERIOD = 10
   val CONSOLE_DEFAULT_UNIT = "SECONDS"
 
   val CONSOLE_KEY_PERIOD = "period"
   val CONSOLE_KEY_UNIT = "unit"
 
-  val pollPeriod = Option(property.getProperty(CONSOLE_KEY_PERIOD)) match {
+  private val pollPeriod = Option(property.getProperty(CONSOLE_KEY_PERIOD)) match {
```

Comment:
Emmm... These logics seems quite similar among different Sinks, could we abstract these logic in `Sink` class, or generate a trait for them?